<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KRGKayitliMusteriler</class>
 <widget class="QDialog" name="KRGKayitliMusteriler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kayıtlı Müşteriler</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="lblKayitliMusteriler">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Kayıtlı Müşteriler</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxAra">
       <property name="title">
        <string>Ara</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,3">
            <item>
             <widget class="QLabel" name="lblAdiSoyadi">
              <property name="text">
               <string>Adı Soyadı</string>
              </property>
              <property name="buddy">
               <cstring>lineEditAdiSoyadi</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditAdiSoyadi"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,6,1,2">
            <item>
             <widget class="QRadioButton" name="radioButtonAliciMusteriler">
              <property name="text">
               <string>Alıcı Müşteriler</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButtonGondericiMusteriler">
              <property name="text">
               <string>Gönderici Müşteriler</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>37</width>
                <height>13</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBoxYazarkenAra">
              <property name="text">
               <string>Yazarken Ara</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonAra">
              <property name="text">
               <string>Ara</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Alıcı Müşteriler</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="tableWidgetAliciListesi"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Gönderici Müşteriler</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="tableWidgetGondericiListesi"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lineEditAdiSoyadi</sender>
   <signal>textChanged(QString)</signal>
   <receiver>KRGKayitliMusteriler</receiver>
   <slot>ara()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>629</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>874</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxYazarkenAra</sender>
   <signal>clicked(bool)</signal>
   <receiver>pushButtonAra</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>953</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>1021</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ara()</slot>
 </slots>
</ui>
